Assets/StreamingAssets/Data/Headlines.xml

```diff
@@ -4,4 +4,17 @@
     <Headline>Notice: Quillcorp has placed an embargo on "Chairs", any Quillcorp Basic Utility Station found in possession of this illegal contraband will be fined 2 million Quillbucks.</Headline>
     <Headline>Introducing the MagBlast 2200 magnetic rifle. 70% of the time, it hits every time.</Headline>
     <Headline>Thinktank finds something that could go wrong. Head of QuillCorp not available for comment.</Headline>
+    <Headline>Attention: Quillcorp has banned all video games except for ProjectPorcupine. Any person caught playing a banned game will face severe penalties.</Headline>
+    <Headline>As the inevitability of his impending death began to crystalise in his mind, Rupert unnacountably wondered whether it would be classified as "W40.9XXD Explosion of unspecified explosive materials (subsequent encounter)" or, he hoped, the more colourful "W59.22XA Struck by turtle (initial encounter)" under the revised and expanded ICD-10-CM American medical billing code scheme.</Headline>
+    <Headline>Attention all Quillcorp customers: Quillcorp security wishes to inform you that everything is okay. There is nothing wrong. Not even a little bit. Please be reassured that staff are doing all that they can to ensure that nothing remains to be amiss, continuing the situation of complete normality and lack of cause for alarm as was previously mentioned.</Headline>
+    <Headline>Quillcorp invites all customers who feel even the slightest bit of unease to please remain calm. Claims that a neurotoxin has been inadvertently released into the air supply are likely to be grossly exaggerated.</Headline>
+    <Headline>Lorem Ipsum</Headline>
```

Comment:
I think this particular news isn't a great example. If you are going to do it, do it right and include at least a few sentences of Lorem Ipsum, but...